src/code_804267C.c

```diff
@@ -279,28 +279,23 @@ void sub_8042A74(Entity *r0)
 
 void sub_8042A84(s16 param_1, Entity *entity, u32 param_3)
 {
-  s32 iVar2;
   s32 iVar3;
   s32 r6;
   s32 param_1_s16;
 
   param_1_s16 = param_1;
 
-  r6 = 0x200;
+  r6 = IntToF248_2(2).raw;
   if (sub_8042768(entity)) {
     sub_804151C(entity,param_1_s16,0);
     sub_803E708(param_3,0x42);
     do {
-      if (0xc7ff < entity->unk1C) break;
+      if (entity->unk1C.raw > IntToF248_2(199.999).raw) break;
```

Comment:
Does >= 200 not work here?

Reply:
For some reason it doesn't, I had to do the same thing in here https://github.com/pret/pmd-red/blob/2d4127bd57fb7f608b6b2f06eb545fe449e1419a/src/code_8069E0C.c#L2229